impl/src/main/java/org/jboss/cdi/tck/tests/event/fires/FireEventTest.java

```diff
@@ -212,8 +211,8 @@ public void testInjectedEventCanHaveBindings() {
     }
 
     /**
-     * This test verifies that binding types can be specified dynamically when firing an event using {@link Event#fire()} by
-     * first using the {@link Event#select()} method to retrieve an Event object with associated binding types.
+     * This test verifies that binding types can be specified dynamically when firing an event using {@code Event#fire()} by
+     * first using the {@code Event#select()} method to retrieve an Event object with associated binding types.
```

Comment:
Not sure why, but probably doesn't hurt.

Reply:
Just do that IDE doesn't complain because in order to use `@link`, you should also specify parameter types which, to me, make the text less readable. So I just went with `@code`, no real difference though :)

Reply:
Ah good point re parameter types, thanks!